crates/zoon/src/style/shadows.rs

```diff
@@ -127,7 +127,7 @@ impl<'a> IntoCowStr<'a> for Shadow {
         if self.inner {
             shadow_settings.push("inset".into())
         }
-        shadow_settings.extend(array::IntoIter::new([
+        shadow_settings.extend(IntoIterator::into_iter([
```

Comment:
`extend` should accept `IntoIterator` (https://doc.rust-lang.org/std/vec/struct.Vec.html#method.extend) so perhaps we can just delete `IntoIterator::inter_iter` altogether and just leave the array `[***]` inside the `extend` call.

Reply:
yes, this one work :)